deploy/vagrant/bin/init_aws.py

```diff
@@ -81,8 +87,12 @@ def configure_aws():
     gen_boto_config(access_key, secret_key)
 
     conn = get_conn()
-    set_security_group(conn, get_ec2_conf()['Security_Group'])
+    ec2conf = get_ec2_conf()
+    if (ec2conf.get('VPC','') == ''):
+      return set_security_group(conn, ec2conf['Security_Group'])
+    else:
+      return set_security_group(conn, ec2conf['Security_Group'], ec2conf['VPC'])
```

Comment:
Since `set_security_group` has considered the condition when `VPC == ''`, the check here seems redundant.